script.js

```diff
@@ -0,0 +1,7 @@
+const humburgerBtn = document.getElementById('humburger-btn')
+    menubar = document.querySelector('ul')
```

Comment:
Kindly avoid using an element tag selector in your javascript quertSelector. It will be better to use a class or ID to avoid selecting the wrong element during implementation